fpdf/fpdf.py

```diff
@@ -3114,6 +3121,50 @@ def _preload_font_styles(self, txt, markdown):
         self.page = page
         return styled_txt_frags
 
+    def _parse_chars(self, txt):
+        "Check if the font has all the necessary glyphs. If a glyph front a fallback font is used, break into fragments"
+        fragments = []
```

Comment:
At the start of this method
`if not self._fallback_fonts:`

then continue a previously to maintain backwards compatibility.